src/components/remotecontrol/remotecontrol.js

```diff
@@ -615,7 +615,7 @@ export default function () {
             lastUpdateTime = now;
             const player = this;
             currentRuntimeTicks = playbackManager.duration(player);
-            updateTimeDisplay(playbackManager.currentTime(player), currentRuntimeTicks);
+            updateTimeDisplay(playbackManager.currentTime(player) * 10000, currentRuntimeTicks);
```

Comment:
We could avoid weird arithmetic by adjusting `playbackManager.duration(player)` to return a time in ms.